<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpDocumentationPage</class>
 <widget class="QWidget" name="HelpDocumentationPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>594</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure help documentation&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line17">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Python Documentation</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="pythonDocDirButton">
        <property name="toolTip">
         <string>Press to select the Python documentation directory via a dialog</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="pythonDocDirEdit">
        <property name="toolTip">
         <string>Enter the Python documentation directory</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="textLabel1_8_2">
        <property name="text">
         <string>&lt;b&gt;Note&lt;/b&gt;: Leave empty to use the PYTHONDOCDIR environment variable, if set.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Qt4 Documentation</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="qt4DocDirButton">
        <property name="toolTip">
         <string>Press to select the Qt4 documentation directory via a dialog</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="qt4DocDirEdit">
        <property name="toolTip">
         <string>Enter the Qt4 documentation directory</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="textLabel1_8_2_2_2">
        <property name="text">
         <string>&lt;b&gt;Note&lt;/b&gt;: Leave empty to use the QT4DOCDIR environment variable, if set.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="title">
      <string>PyQt4 Documentation</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="pyqt4DocDirButton">
        <property name="toolTip">
         <string>Press to select the PyQt4 documentation directory via a dialog</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="pyqt4DocDirEdit">
        <property name="toolTip">
         <string>Enter the PyQt4 documentation directory</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="textLabel1_8_2_2_3">
        <property name="text">
         <string>&lt;b&gt;Note&lt;/b&gt;: Leave empty to use the PYQT4DOCDIR environment variable, if set.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="pysideGroup">
     <property name="title">
      <string>PySide Documentation</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="pysideDocDirButton">
        <property name="toolTip">
         <string>Press to select the PySide documentation directory via a dialog</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="pysideDocDirEdit">
        <property name="toolTip">
         <string>Enter the PySide documentation directory</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="textLabel1_8_2_2">
        <property name="text">
         <string>&lt;b&gt;Note&lt;/b&gt;: Leave empty to use the PYSIDEDOCDIR environment variable, if set.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>479</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pythonDocDirEdit</tabstop>
  <tabstop>pythonDocDirButton</tabstop>
  <tabstop>qt4DocDirEdit</tabstop>
  <tabstop>qt4DocDirButton</tabstop>
  <tabstop>pyqt4DocDirEdit</tabstop>
  <tabstop>pyqt4DocDirButton</tabstop>
  <tabstop>pysideDocDirEdit</tabstop>
  <tabstop>pysideDocDirButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
